content/millennium/r4/medications/medication-request.md

```diff
@@ -0,0 +1,137 @@
+   * [Patient Instructions](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.patientInstruction){:target="_blank"}
+   * [Timing Schedule](https://hl7.org/fhir/r4/datatypes-definitions.html#Timing){:target="_blank"}
+   * [As Needed](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.asNeeded_x_){:target="_blank"}
+   * [Body Site](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.site ){:target="_blank"}
+   * [Route](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.route){:target="_blank"}
+   * [Dose and Rate](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.doseAndRate){:target="_blank"}
+* [Dispensing Details](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest){:target="_blank"}
+    * [Validity Period](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.validityPeriod ){:target="_blank"}
+    * [Number of Refills](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.numberOfRepeatsAllowed){:target="_blank"}
+    * [Dispense Quantity](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.quantity){:target="_blank"}
+    * [Expected Days Supply per Dispense](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.expectedSupplyDuration){:target="_blank"}
+    * [Intended Dispenser](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.performer){:target="_blank"}
+* [Substitution Allowed Boolean](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.substitution.allowed_x_){:target="_blank"}
+* [Prior Prescription](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.priorPrescription){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:medication_request, :r4) %>
+
+### Contained Medication Bindings
+
+<%= terminology_table(:medication_request_contained_medication, :r4) %>
+
+## Search
+
+Search for MedicationRequests that meet supplied query parameters:
+
+    GET /MedicationRequest?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name      | Required?          | Type          | Description
+-----------|--------------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource id associated with the resource. Example: `12345`
+ `patient`                | This, or `_id`     | [`reference`] | The specific patient to return MedicationRequests for. Example: `12345`
+ `status`                 | N                  | [`token`]     | The status of the medication, may be a list separated by commas. Example: `active,completed`
```

Comment:
should we say anything about the system being provided?

Reply:
Hopefully a link will suffice: https://github.com/cerner/fhir.cerner.com/pull/332/commits/7bee2124b41c1252ae9347951199fbd8db627be3#diff-91cabf44fbcf0c5aaa861d340e57d246R75